packages/react-components/react-table/stories/Table/CellNavigation.stories.tsx

```diff
@@ -108,3 +108,15 @@ export const CellNavigation = () => {
     </Table>
   );
 };
+
+CellNavigation.parameters = {
```

Comment:
not related to the PR, but the fact that storybook shows only the exported component source code makes the examples really hard to read - I do not know what the `columns` and `items` are, where you import `useArrowNavigationGroup` from...  :/

Reply:
yeah we've had an open issue on it for a while #20046